### PR TITLE
Use KDE upstream for 5.15 live builds.

### PR DESCRIPTION
**Pull request by Chiitoo**
What do we think about this?

Should we wait and see for a while longer, or go ahead and make use of it?

### CODE REVIEW

**Comment by a17r** [comment]
https://mail.kde.org/pipermail/distributions/2021-April/000984.html
https://dot.kde.org/2021/04/06/announcing-kdes-qt-5-patch-collection
https://community.kde.org/Qt5PatchCollection

File: eclass/qt5-build.eclass [hunk 1]
```diff
@@ -67,10 +67,11 @@ QT5_MINOR_VERSION=$(ver_cut 2)
 readonly QT5_MINOR_VERSION
 
 case ${PV} in
-	5.??.9999)
-		# git stable branch
+	5.15.9999)
+		# KDE upstream for 5.15 patches
+		HOMEPAGE+="https://invent.kde.org/qt/qt/${QT5_MODULE}/-/tree/kde/5.15"
```
Comment (Pesa) [inline eclass/qt5-build.eclass:72]: I don't think HOMEPAGE can contain variables[1]. Maybe just point to https://invent.kde.org/qt/qt ?

[1] https://devmanual.gentoo.org/ebuild-writing/variables/index.html#ebuild-defined-variables

Comment (Chiitoo) [reply eclass/qt5-build.eclass]: Ah, right.  Completely forgot about that.

Changed as suggested.

File: eclass/qt5-build.eclass [hunk 2]
```diff
@@ -67,10 +67,11 @@ QT5_MINOR_VERSION=$(ver_cut 2)
 readonly QT5_MINOR_VERSION
 
 case ${PV} in
-	5.??.9999)
-		# git stable branch
+	5.15.9999)
+		# KDE upstream for 5.15 patches
+		HOMEPAGE+="https://invent.kde.org/qt/qt/"
```
Comment (a17r) [inline eclass/qt5-build.eclass:72]: HOMEPAGE Needs a space between URIs ;)

Sorry re variable, didn't think of that myself.

Comment (Chiitoo) [reply eclass/qt5-build.eclass:72]: Crud!  Another thing I've made note in the past, and completely didn't think of now, of course.

**Review by Pesa — APPROVED** [review]

**Review by a17r — APPROVED** [review]

**Pull request merged**